contrib/samples/sample.neomuttrc-tlr

```diff
@@ -194,12 +194,12 @@ set   abort_unmodified=no               # Let me send empty messages
 set   alias_file=~/.mutt/aliases        # Where to store aliases
 unset allow_8bit                        # Produce correct MIME
 unset arrow_cursor                      # Use the bar cursor
-set   askcc                             # Ask me about CCs
+set   ask_cc                             # Ask me about CCs
```

Comment:
perhaps you should try to keep the `#` aligned

Reply:
Apologies for that! It's resolved now. :-)